scripts/rhel_validation/validate_rhel_vm.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -xoe pipefail
+
+ret_val() {
+    while read val;do
+        last_val="$val"
+    done
+    echo "$last_val"
+}
+
+run_remote_commands() {
+    PRIVATE_KEY="$1"
+    USERNAME="$2"
+    REMOTE_IP="$3"
+    COMMANDS="$4"
+
+    IFS=';'; COMMANDS=($COMMANDS); unset IFS;
+    for comm in "${COMMANDS[@]}"; do
+        trimmed_com="$(echo $comm | xargs)"
+        output="$(ssh -i "$PRIVATE_KEY" -o StrictHostKeyChecking=no -o ConnectTimeout=10 "$USERNAME@$REMOTE_IP" "$trimmed_com")"
+        echo "$output"
+    done
+}
+
+wait_for_ip() {
+    FULL_BUILD_NAME="$1"
+    RESOURCE_GROUP="$2"
+
+    COUNTER=0
+    INTERVAL=5
+    AZURE_MAX_RETRIES=60
+
+    while [ $COUNTER -lt $AZURE_MAX_RETRIES ]; do
+        public_ip_raw=$(az network public-ip show --name "$FULL_BUILD_NAME-PublicIP" --resource-group kernel-validation --query '{address: ipAddress }')
```

Comment:
line too long and this is similar to the one from the validate azure vm, can it be reused?